js/lbank2.js

```diff
@@ -444,13 +444,14 @@ module.exports = class lbank2 extends Exchange {
          */
         await this.loadMarkets ();
         const market = this.market (symbol);
+        if (limit === undefined) {
```

Comment:
Is there a default value that we could assign to the `limit` variable (if it's required, but not specified) ?

Reply:
@kroitor how about 60? 
![image](https://user-images.githubusercontent.com/87606888/169885672-2ca8f6e3-3597-4d5b-ade0-1a1a5059ca50.png)

Reply:
Sounds good!